### ISSUE TITLE
Lightbox image zoomed in from carousel

### ISSUE THREAD

**Issue by krissl94** [vvvmax/unitegallery#72]
Hi there!

When clicking on an item in the carousel to display the lightbox, the image is zoomed to 100% width of the screen..  I am working with version 1.7.18.

**Comment by vvvmax** [comment]
yes, the wide lightbox is opens 100%. if you want, you can change to
compact lightbox, then it will be a bit smaller.

**Comment by krissl94** [comment]
If this is supposed to happen, why doesn't it happen with other themes?

**Comment by vvvmax** [comment]
HI. I need to see what you talking about. right now I don't understand.
Please write me a ticket: valiano.ticksy.com
send me a link to your gallery so I could look.
if it some bug, then I'll try to fix it.

**Comment by krissl94** [comment]
You can see the bug occuring [here](http://tharprojects.nl/edityourweb/carouselexample).
When you click on an image, the lightbox is opened with the image zoomed in.

**Comment by vvvmax** [comment]
HI. you have too many options here:

Common.initializeGalleries = function(){

in this file:

http://tharprojects.nl/edityourweb/js/common.js?1462622348

I can't understand what option exactly affecting scale mode.

Please run the gallery without all the modifications and check.

then return them one by one.
